UnitsNet/GeneratedCode/Quantities/Acceleration.NetFramework.g.cs

```diff
@@ -738,9 +745,6 @@ public double As(AccelerationUnit unit)
             return Convert.ToDouble(converted);
         }
 
-        /// <inheritdoc />
-        public double As(Enum unit) => As((AccelerationUnit) unit);
```

Comment:
Removed as there's no point in exposing the As(Enum) method on the class itself. People would just use As(AccelerationUnit) on an Acceleration instance. It's really only for IQuantity, so just keeping the explicit interface implementation.

Reply:
Good catch